Frameworks/UIKit/UIBezierPath.mm

```diff
@@ -190,75 +152,74 @@ - (void)addArcWithCenter:(CGPoint)center
               startAngle:(CGFloat)startAngle
                 endAngle:(CGFloat)endAngle
                clockwise:(BOOL)clockwise {
-    CGPathAddArc(_path, NULL, center.x, center.y, radius, startAngle, endAngle, clockwise == FALSE);
+    CGPathAddArc(_workingPath, nullptr, center.x, center.y, radius, startAngle, endAngle, clockwise == TRUE);
 }
 
 /**
  @Status Interoperable
 */
 - (void)addLineToPoint:(CGPoint)pt {
-    CGPathAddLineToPoint(_path, getTransform(self), pt.x, pt.y);
+    CGPathAddLineToPoint(_workingPath, nullptr, pt.x, pt.y);
 }
 
 /**
  @Status Interoperable
 */
 - (void)moveToPoint:(CGPoint)pt {
-    CGPathMoveToPoint(_path, getTransform(self), pt.x, pt.y);
+    CGPathMoveToPoint(_workingPath, nullptr, pt.x, pt.y);
 }
 
 /**
  @Status Interoperable
 */
 - (void)closePath {
-    CGPathCloseSubpath(_path);
+    CGPathCloseSubpath(_workingPath);
 }
 
 /**
  @Status Interoperable
 */
 - (CGPathRef)CGPath {
-    return _path;
+    return (CGPathRef)[(id)CGPathCreateCopy(_workingPath) autorelease];
 }
 
 /**
  @Status Interoperable
 */
 - (void)setCGPath:(CGPathRef)path {
-    CGPathRetain(path);
-    CGPathRelease(_path);
-    _path = path;
+    _workingPath = woc::MakeStrongCF(CGPathCreateMutableCopy(path));
```

Comment:
The way it is now, it creates a temporary StrongCF, copies it (+1) to _workingPath, then destructs it (-1)... If you use `attach` you get a true `+0`